pom.xml

```diff
@@ -580,7 +608,7 @@
                     <limit>
                       <counter>LINE</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>0.89</minimum>
+                      <minimum>0.87</minimum>
```

Comment:
what is the reason to reduce it?
@romani wasn't the problem that SQ needs a passed quality gate and they also check the coverage?

Reply:
My bad, I have rectified it.